src/test/java/org/apache/onami/persist/test/transaction/testframework/TransactionalWorker.java

```diff
@@ -109,7 +109,7 @@ public void doTasks() {
   /**
    * Check all stored entities if they actually have been persisted in the DB.
    */
-  @Transactional
+  @Transactional(onUnits = {})
```

Comment:
Also we still need to support un-annotated entity managers, only thing we are requiring is explicitly saying that in the annotation.